### PR TITLE
Readme

### LINKED ISSUE #12: Add file tree to ReadMe

### PR DESCRIPTION
**Pull request by Arique1104**
This adds:
- A readme with preselected sections
- Pipfile
- .gitignore file

closes #12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
...
+
+## Visual Schema
+
+## Tech stack
+
+## Development Database Instructions
+
+## Endpoint Documentation
+
+## File Tree
```
Comment (priyapower) [inline README.md:18]: I like that this is here for now, once we complete the MVP server we can update this to an image

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
-# DreamHome_BEE
\ No newline at end of file
+# DreamHome BackEnd
```
Comment (priyapower) [inline README.md:1]: Lovely initial Readme

File: dreamhome.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from flask import Flask
```
Comment (priyapower) [inline dreamhome.py:1]: Great initial runner - we may want to call this file `app.py` (convention, not an error)

**Review by priyapower — APPROVED** [review]
Looks great!

**Review by timkeresey — COMMENTED** [review]
This will make it easy for us to reference necessary endpoints, thank you!

**Pull request merged**